src/components/cluster/cluster.scss

```diff
@@ -0,0 +1,31 @@
+.cluster-modal-wrapper {
+    display: grid;
+    grid-template-columns: auto 1fr;
+}
+.cluster-modal-number {
+    border-radius: 50%;
+    z-index: 2;
+}
+
+.cluster-modal-container {
+    width: 404px;
+    height: 120px;
+}
+
+.cluster-inner-container {
+    margin-left: -28px;
+}
+
+.cluster-inner-container.cluster-last-item {
+    border-left: none;
+    padding-bottom: 0;
+}
+
+.default-white .tippy-content {
```

Comment:
We should not overwrite default property of component directly, we should wrap it in some parent css

Reply:
Done